bin/final_generate_v75_gene_bed.R

```diff
@@ -3,15 +3,16 @@
 # __author__      = "Alexandria Dymun"
 # __email__       = "pintoa1@mskcc.org"
 # __contributor__ = "Anne Marie Noronha (noronhaa@mskcc.org)"
-# __version__     = "0.0.1"
+# __version__     = "0.0.2"
 # __status__      = "Dev"
 
 
 suppressPackageStartupMessages({
-#    library(plyr)
+    library(plyr)
     library(dplyr)
     library(data.table)
     library(stringr)
+    options(scipen = 999)
```

Comment:
what's this?

Reply:
When I was generating the gene bed, the start positions were being put in sceinticif format if e+7 with only two decimal places for some reason.